### PR TITLE
835 allow null lat & long

### PR DESCRIPTION
**Pull request by hwindsor**
## Pull Request Overview

- allow null for decimalLatitude and decimalLongitude but only if both of them are null

---

### CODE REVIEW

**Review by brmscheiner — APPROVED** [review]

**Review by colinwkingen — APPROVED** [review]
LGTM

File: app/modules/asset_groups/metadata.py [hunk 1]
```diff
@@ -137,6 +137,22 @@ def _validate_lat_long(cls, dictionary, error_str):
...
+        if (
+            'null' == dictionary['decimalLatitude']
+            and 'null' == dictionary['decimalLongitude']
+        ):
+            return
+        if (
+            'null' == dictionary['decimalLatitude']
+            or 'null' == dictionary['decimalLongitude']
+        ):
+            raise AssetGroupMetadataError(
```
Comment (naknomum) [inline app/modules/asset_groups/metadata.py:151]: pretty sure these `'null'` comparisons need to instead be comparing against `None`, as that is what the json `{"decimalLatitude": null, ...}` gets deserialized to.   thus, more like `if not dictionary['decimalLatitude']...`

**Review by naknomum — CHANGES_REQUESTED** [review]
can someone let me know if i am wrong about the `'null`' vs `None` stuff?

File: app/modules/asset_groups/metadata.py (hunk 1 above)
Comment (hwindsor) [reply app/modules/asset_groups/metadata.py:151]: > pretty sure these `'null'` comparisons need to instead be comparing against `None`, as that is what the json `{"decimalLatitude": null, ...}` gets deserialized to. thus, more like `if not dictionary['decimalLatitude']...`

exactly what I just found and why this is draft again

**Comment by hwindsor** [comment]
> can someone let me know if i am wrong about the `'null`' vs `None` stuff?

You were right, now fixed

**Review by naknomum — APPROVED** [review]
all good now

**Pull request merged**